workflow-core/src/main/java/com/squareup/workflow1/Tracing.kt

```diff
@@ -0,0 +1,85 @@
+package com.squareup.workflow1
+
+/**
+ * Tracing facade to install later.
+ *
+ * This follows the static service locator that Timber demonstrates.
+ * See [Timber][https://github.com/JakeWharton/timber/blob/trunk/timber/src/main/java/timber/log/Timber.kt].
+ */
+public class Tracing private constructor() {
+
+  public interface Tracer {
+    public fun beginIdSection(label: () -> WorkflowIdentifier)
+    public fun endIdSection(label: () -> WorkflowIdentifier)
```

Comment:
I'm not sure I understand how this is going to be implemented. Do you have an example? It looks like this PR does not have any actual tracing code hooked up?

Reply:
It does. There is tracing hooked up in the complex-poetry app.